scripts/test.js

```diff
@@ -112,4 +113,16 @@ executor.addCommandlineTest(
       message: 'warning -- Module "xApp" was created but never loaded.'
     }]);
 
+// Check accessibility plugin
+
+executor.addCommandlineTest(
+    'node lib/cli.js plugins/accessibility/spec/failureConfig.js')
+    .expectExitCode(1)
+    .expectErrors([{
+      message: 'warning -- accessibility plugin cannot be run as there is ' +
```

Comment:
Great idea. I'll do that! The rest of the feedback has been incorporated. Thanks!